### PR TITLE
[Feature Request]: Email on Contact Us

### LINKED ISSUE #334: [Feature Request]:  Email on Contact Us
### Is there an existing issue for this?

- [X] I have searched the existing issues

### Feature Description

If i want to implement to the `Contact us` page.

## **What Feature are add the Contact Us page .**
-  If the user can be suggestion to give `Example : This is project is super` in this content type to send also `user` mail address we want to implement the return to send `Thank you for contacting..` in user mail address.

## How to Implement
-  **Backend**
-  Nodemailer and Mongodb

## How to Implement NodeMailer
```
async function sendOTPEmail(email, otp) {
    let transporter = nodemailer.createTransport({
        service: 'gmail',  // Adjust this according to your email service
        auth: {
            user: 'your-email@gmail.com',
            pass: 'your-email-password'
        }
    });

    let info = await transporter.sendMail({
        from: '"OTP Service" <your-email@gmail.com>',
        to: email,
        subject: 'Your OTP Code',
        text: `Your OTP code is: ${otp}`
    });

    console.log('Message sent: %s', info.messageId);
}
```

### Add ScreenShots

_No response_

### Priority

High

### Record

- [X] I have read the Contributing Guidelines
- [X] I'm a GSSOC'24 contributor
- [X] I have starred the repository

### PR DESCRIPTION
**Pull request by sivaprasath2004**
# Pull Request

### Title
[Feature Request]: Email on Contact Us

### Description
The mail will be Announced.

### Related Issues
Fixes #334 

### Changes Made
- Improvements

### Checklist 

- [ ] I have tested the changes locally
- [x] Documentation has been updated (if necessary)
- [ ] Changes are backward-compatible

### Screenshots (if applicable) 

[Screencast from 2024-07-03 20-05-18.webm](https://github.com/VaibhavArora314/StyleShare/assets/121082414/eb4c91fa-d94c-437f-b887-d292d4615d5f)

WhatsApp Image 2024-07-03 at 8 06 59 PM

### Additional Notes

Footer

### CODE REVIEW

**Comment by sivaprasath2004** [comment]
@VaibhavArora314 i will finished it ...
please merge..

**Review by VaibhavArora314 — CHANGES_REQUESTED** [review]
as I stated in the issue, also add content of the contact us form filled by the user in the email

File: backend/src/helpers/mail/ContactUsMail.ts [hunk 1]
```diff
@@ -0,0 +1,60 @@
...
+            <p style="font-family: monospace;font-weight: 700;">Keep connected with our site <a href="https://style-share.vercel.app/app">https://style-share.vercel.app/app</a>...🚀 for updates and more style tips!</p>
+<br><br>
+    <p style="font-family: monospace;text-align: center;">Happy Coding :)  Happy Journey :)</p>
+        </div>
+    </main>
+</body>
+</html>
+
+`
+export const mailing=async (email: string, username: string) =>{
```
Comment (akbatra567) [inline backend/src/helpers/mail/ContactUsMail.ts:34]: Username is unused. could you mention what was the intention behind it ?

File: backend/src/helpers/mail/ContactUsMail.ts [hunk 2]
```diff
@@ -0,0 +1,60 @@
...
+const sender=nodeMailer.createTransport({
+    service:"Gmail", 
+    auth:{
+        user: process.env.EMAIL_USER,
+        pass: process.env.EMAIL_PASS,
+    },
+})
+const html= HTML
+const receiver={
+    from:'no_reply<yourapp@example.com>',
```
Comment (akbatra567) [inline backend/src/helpers/mail/ContactUsMail.ts:44]:
```suggestion
    from: process.env.FROM_EMAIL,
```

make this configurable from the environment variables.

**Comment by sivaprasath2004** [comment]
Hello @VaibhavArora314 , @akbatra567 

- I will finish it...

WhatsApp Image 2024-07-04 at 7 07 27 AM

**Review by VaibhavArora314 — COMMENTED** [review]
increase the font size of message, rest looks good
image

**Comment by sivaprasath2004** [comment]
@VaibhavArora314  which one increase `Your Message`

**Comment by sivaprasath2004** [comment]
Hello @VaibhavArora314 

Screenshot from 2024-07-05 07-07-26

changed it...

**Review by VaibhavArora314 — APPROVED** [review]

**Review by Ultimateutkarsh11 — APPROVED** [review]

**Pull request merged**